### DISCUSSION TITLE
Initiative

### DISCUSSION THREAD

**Discussion by t-luethi** [fvtt-fria-ligan/forbidden-lands-foundry-vtt#15] [Q&A] [answered]
Hey!
When rolling for initiative in the FL-system on Foundry VTT, it sometimes happens that two or more actors roll the same number. When this happens, the initiative for those actors just follows an alphabetical order. E.g. ‘Player A’ and ‘Player B’ both roll a ‘5’ – in this case ‘Player A’ will always go first. That does not seem to be intended; is there a possbility to change this behavior?
Cheers

**Comment by aMediocreDad** [comment, marked as answer]
Hey there @t-luethi 😸 This is unfortunately down to how the initiative system works in Foundry. It can be altered however, though it is some work to get done. I have set it up as an issue and it is definitely something we want to look at as it's a core part of the system. 

For now I would suggest you make a rolltable with entries 1-10 then uncheck draw with replacement. If you have the cards from the Free League website you can even use them as images for the results. You can then let players roll on the table and manually input the numbers in the initiative tracker. (It's some work, but it's a work around).

Alternately you can look at my [macro module](https://github.com/aMediocreDad/fbl-macros) which illustrates how to use the [unofficial card support module](https://foundryvtt.com/packages/cardsupport/). I use it in a very manual way. Like so:
![2020-11-19](https://user-images.githubusercontent.com/9851733/106503129-8c4b1d80-64c5-11eb-8cae-084346d4d23b.png)

Hope this helped a bit 🙂

**Reply by t-luethi** [reply]
Thanks for the reply, that helped a lot!
